presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveStorageFormats.java

```diff
@@ -465,6 +546,21 @@ private void setAdminRole(Connection connection)
         }
     }
 
+    /**
+     * Ensures that a view named "dummy" with exactly one row exists in the default schema.
+     */
+    // These tests run on versions of Hive (1.0.0 on CDH 5) that don't fully support SELECT without FROM
+    // (particularly when using certain forms of authentication).
```

Comment:
Is it related to authentication?

Reply:
The original issue was that Hive tries to check permissions on the internal dummy table when you use a bare `SELECT`, and fails. I'm not actually sure if it was related to some types of authentication or if it shows up with any authentication at all.

(It looks like bug slipped through the gaps in Hive's development process. The [ticket](https://issues.apache.org/jira/browse/HIVE-10625) for this bug in version 1.1 was closed because it was resolved in [another ticket](https://issues.apache.org/jira/browse/HIVE-11498), but that one doesn't list 1.1.)